Unix/http/http.c

```diff
@@ -1477,7 +1477,11 @@ static MI_Result _CreateSSLContext(Http* self, const char* sslCipherSuite, Serve
     {
         options |= SSL_OP_NO_SSLv3;
     }
-    if ( SSL_CTX_set_options(sslContext, options) == 0 )
+#if defined(linux)
```

Comment:
not much point #ifdef-ing this as it hits most platforms. I thought this was just there for solaris or something like that.